service/history/nDCBranchMgr.go

```diff
@@ -18,6 +18,8 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+//go:generate mockgen -copyright_file ../../LICENSE -package $GOPACKAGE -source $GOFILE -destination nDCBranchMgr_mock.go
```

Comment:
Add `go generate` to Makefile?

Reply:
not really, this should be generated manually

if interface is changed, then the test should not work at all